plugins/nginx_logs.yaml

```diff
@@ -0,0 +1,129 @@
+        operators:
+          {{ if eq .log_format "default" }}
+          - type: regex_parser
+            regex: '^(?P<http_request_remoteIp>[^ ]*) (?P<host>[^ ]*) (?P<user>[^ ]*) \[(?P<time_local>[^\]]*)\] "(?P<http_request_requestMethod>\S+)(?: +(?P<http_request_requestUrl>[^\"]*?)(?: +(?P<http_request_protocol>\S+))?)?" (?P<http_request_status>[^ ]*) (?P<http_request_responseSize>[^ ]*)(?: "(?P<http_request_referer>[^\"]*)" "(?P<http_request_userAgent>[^\"]*)")?$'
+            timestamp:
+              parse_from: "$body.time_local"
+              layout: '%d/%b/%Y:%H:%M:%S %z'
+            severity:
+              parse_from: "$body.http_request_status"
+              mapping:
+                info: 2xx
+                info2: 3xx
+                warn: 4xx
+                error: 5xx
+          {{end}} # End default log format parsing
+          
+          {{ if eq .log_format "observiq" }}
+          - id: access_parser
+            type: json_parser
+            timestamp:
+              parse_from: "$body.time_local"
+              layout: '%d/%b/%Y:%H:%M:%S %z'
+            severity:
+              parse_from: "$body.status"
+              preserve_to: "$body.http_request_status"
+              mapping:
+                info: 2xx
+                info2: 3xx
+                warn: 4xx
+                error: 5xx
+          - id: request_parser
+            type: regex_parser
+            parse_from: "$body.request"
+            preserve_to: "$body.request"
+            if: '$body.request != nil and $body.request matches "\\S+ +[^ ]* "'
+            regex: '(?P<method>\S+) +(?P<path>[^ ]*) ((?P<protocol>[^/]*)/(?P<protocol_version>.*)|.*)?'
+          {{end}} # end observiq format parsing
+      {{end}} # end access log receiver
+
+      {{- if .enable_error_log -}}
+      filelog/error_log:
+        include:
+          {{ range $fp := .error_log_paths }}
+          - '{{ $fp }}'
+          {{end}}
+        attributes:
+          log_type: 'nginx.error'
+        start_at: {{ .start_at }}
+        encoding: {{ .encoding }}
+        operators:
+          - type: regex_parser
+            regex: '^(?P<time>\d+[./-]\d+[./-]\d+[- ]\d+:\d+:\d+) \[(?P<log_level>[^\]]*)\] (?P<pid>\d+)#(?P<tid>\d+):(?: \*(?P<connection>\d+))? (?P<message>.*?)(?:, client: (?P<client>[^,]+))?(?:, server: (?P<server>[^,]+))?(?:, request: "(?P<request>[^"]*)")?(?:, subrequest: \"(?P<subrequest>[^\"]*)\")?(?:, upstream: \"(?P<upstream>[^"]*)\")?(?:, host: \"(?P<host>[^\"]*)\")?(?:, referrer: \"(?P<referer>[^"]*)\")?$'
+            timestamp:
+              parse_from: "$body.time"
+              layout: '%Y/%m/%d %T'
+            severity:
+              parse_from: "$body.log_level"
+              mapping:
+                info: 2xx
+                info2: 3xx
+                warn: 4xx
+                error: 5xx
+      {{end}} # end error log receiver
+
+    service:
+      pipelines:
+        logs:
+          receivers:
+          {{ if .enable_access_log }}
```

Comment:
Does this need to be indented one?

Reply:
indented seems more consistent with other plugins, good catch